### PR TITLE
[Merged by Bors] - refactor: use CdfToKernel in CondCdf

### PR DESCRIPTION
**Pull request by RemyDegenne**
The recently added file CdfToKernel contains generalizations of many lemmas in the file CondCdf. This PR refactors CondCdf to use the new results.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by sgouezel** [comment]
Could you add a comment in the file-level docstring how most things are deduced from the more general ` CdfToKernel`, and how the notions are related?

File: Mathlib/Probability/Kernel/Disintegration/CondCdf.lean [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+The construction of the conditional cdf in this file follows the proof of Theorem 3.4 in
+[O. Kallenberg, Foundations of modern probability][kallenberg2021].
+
+-/
+
+open MeasureTheory Set Filter TopologicalSpace
+
+open scoped NNReal ENNReal MeasureTheory Topology
+
+section AuxLemmasToBeMoved
```
[the remark this answers is not in the document]
Comment (RemyDegenne) [reply Mathlib/Probability/Kernel/Disintegration/CondCdf.lean]: I found out there is another PR open about removing those lemmas: #10298
Given that they should be deleted soon, I moved them to CdfToKernel to be able to use them in that file, but did not look for a better place.

File: Mathlib/Probability/Kernel/Disintegration/CondCdf.lean [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+
+namespace ProbabilityTheory
+
+variable {α β ι : Type*} {mα : MeasurableSpace α}
+
+attribute [local instance] MeasureTheory.Measure.IsFiniteMeasure.IicSnd
+
+/-! ### Auxiliary definitions
+
+We build towards the definition of `ProbabilityTheory.cond_cdf`. We first define
```
Comment (sgouezel) [inline Mathlib/Probability/Kernel/Disintegration/CondCdf.lean:139]:
```suggestion
We build towards the definition of `ProbabilityTheory.condCDF`. We first define
```

**Review by sgouezel — COMMENTED** [review]
Looks great, thanks!
bors d+

**Pull request closed without merging**